settings.py

```diff
@@ -180,8 +187,23 @@
 AUTHENTICATION_BACKENDS = (
     'django.contrib.auth.backends.ModelBackend',
     'guardian.backends.ObjectPermissionBackend',
+    'allauth.account.auth_backends.AuthenticationBackend',
```

Comment:
so this looks like we support multiple auth backends - is that the intention here?

Reply:
The `django.contrib.auth.backends.ModelBackend` is always required event for Google Auth. I'm not sure about `guardian.backends.ObjectPermissionBackend`